easybuild/framework/extension.py

```diff
@@ -111,6 +111,11 @@ def __init__(self, mself, ext, extra_params=None):
         # construct dict with template values that can be used
         self.cfg.template_values.update(template_constant_dict({'name': name, 'version': version}))
 
+        # Add install/builddir templates, copied from update_config_template_run_step
+        # do we need to call self.cfg.generate_template_values() after?
```

Comment:
No need to call `generate_template_values`, so you can drop this comment.

Reply:
I thought so, since that's already done in master.